apps/socket/src/rooms/tic-tac-toe.room.ts

```
@@ -0,0 +1,121 @@
+			if (this.board[cell]) {
+				throw new Error("Cell is already taken");
+			}
+
+			if (cell < 0 || cell > 8) {
+				throw new Error("Invalid cell");
+			}
+
+			this.board[cell] = this.getCurrentPlayer()?.symbol!;
+			this.nextTurn();
+		} catch (error) {
+			throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)